Podfile

```diff
@@ -5,6 +5,8 @@ yajlVersion='~> 2.0.4'
 
 target 'AdblockPlusSafari' do
 
+pod 'yajl', $yajlVersion
```

Comment:
Why is this necessary? How I see it, yajl is currently not being used in the host app.